src/main/java/io/debezium/connector/vitess/connection/OtherMessage.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.vitess.connection;
+
+import java.time.Instant;
+import java.util.List;
+
+/** Whether this message represents an OTHER event.*/
```

Comment:
What is an "OTHER" event?

Reply:
This is an empty event that VStream sends during RESHARD operation and MOVETABLE operation. For example, the following is what VStream sends to the client during resharding:
```
type: VGTID
vgtid {
  shard_gtids {
    keyspace: "customer"
    shard: "-80"
    gtid: "MySQL56/3ed127eb-d883-11ea-b63b-0242ac110010:1-7"
  }
}

type: OTHER
timestamp: 1596788604 (the approx time when reshard is run)
current_time: 1596788604923504075
```
We're ignoring `OTHER` event for now. This means, in the above response sent by VStream, we won't create any `SourceRecord` to Kafka.